### DISCUSSION TITLE
Issue Installing Java Env

### DISCUSSION THREAD

**Discussion by dreamness-dnalm** [rg35xx-cfw/rg35xx-cfw.github.io#99] [Q&A]
I am encountering an issue while attempting to install Java on the OS. Despite following standard installation procedures for OpenJDK and Azul Zulu JDK, I am faced with an execution error.   
Below is the error message I receive:
```text
-bash: ./bin/java: cannot execute: required file not found
```
version: 20240126

reference
https://wiki.batocera.org/systems:ports

thank you

**Comment by usmanshabbir756** [comment]
In Search, search for and then select: System (Control Panel)
Click the Advanced system settings link.
Click Environment Variables. In the section System Variables find the PATH environment variable and select it. Click Edit. If the PATH environment variable does not exist, click New.
In the Edit System Variable (or New System Variable) window, specify the value of the PATH environment variable. Click OK. Close all remaining windows by clicking OK.
Reopen Command prompt window, and run your java code.
